seqr/views/utils/pedigree_info_utils_2_3_tests.py

```diff
@@ -26,16 +27,17 @@ def test_parse_pedigree_table(self):
             [['family_id', 'individual_id', 'sex', 'affected', 'father', 'mother'],
              ['', '', 'male', 'u', '.', 'ind2']], FILENAME)
         self.assertListEqual(records, [])
-        self.assertListEqual(
-            errors, ["Error while converting {} rows to json: Family Id not specified in row #1:\n{{'affected': 'u', 'maternalId': 'ind2', 'individualId': '', 'sex': 'male', 'familyId': '', 'paternalId': ''}}".format(FILENAME)])
+        errors = errors[0].split('\n')
```

Comment:
you need to keep testing that there is only 1 error

Reply:
Added